MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+module(
+    name = "io_bazel_rules_go",
+    version = "0.33.0",
+    compatibility_level = 1,
```

Comment:
Is it okay to put the compatibility level to be the same as the major version, `0`?

Reply:
It does work.